packages/cf-component-card/src/CardBlock.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  padding: '1rem 1.4rem 1.26667rem',
+  borderTop: `1px solid ${theme.colorGrayLight}`,
```

Comment:
Yeah but then I have the `&:first-child` selector below that makes sure that the border is `0` on the first child similar as the previous CSS rule.